### ISSUE TITLE
Clarify how Tokei counts blank lines inside comment blocks

### ISSUE THREAD

**Issue by Luni-4** [XAMPPRocky/tokei#498]
`Tokei` counts blank lines inside comment blocks as blank lines. Wouldn't they be considered as comment lines?

For example:
```c
/* Copyright (C) 2007-2008 Foo
   Copyright (C) 2008      Foo

   File: test.c
  
*/
```

`Tokei` output: 
```
-------------------------------------------------------------------------------
 Language            Files        Lines         Code     Comments       Blanks
-------------------------------------------------------------------------------
 C                     1            6            0            4            2
-------------------------------------------------------------------------------
 Total                 1            6            0            4            2
-------------------------------------------------------------------------------
```
In this case, `Blanks` should be 0, while `Comments` 6.

Thanks in advance for your clarification! :)

**Comment by foxpy** [comment]
What is the point of blank line inside of a comment? I believe it makes more sense to count these as blanks.

**Comment by Luni-4** [comment]
Yep, I can agree with you, but I was just looking at that from a `parent-child` perspective. Since the blank line is inside a comment block, it is indeed a comment. Furthermore there are metrics, like the `LOCbl`, that counts the number of blank lines in a file, but it excludes the blank lines inside a comment block because they are considered to be a comment line. That's why I was asking for a clarification :)

**Comment by XAMPPRocky** [comment]
Thank you for your issue! I can definitely see the of why one could see it as comments in this case. I haven't really given much thought on whether it should be or the other.

**Comment by Luni-4** [comment]
Thanks to you for accepting my issue! 
In order to not break the current implementation, perhaps we could add a cli option, something like: `--blank-as-comment-block`.

**Comment by XAMPPRocky** [comment]
I would be hesitant about making an option, I want to keep the amount of bespoke behaviour flags to a minimum. I would prefer to change the implementation if I was going to make a change.

**Comment by foxpy** [comment]
I really think that current implementation makes much more sense. When counting the number of comment lines, I will possibly like to know how much of the comments are meaningful.
